### PR TITLE
Remove layer_selector plugin

### PR DESCRIPTION
**Pull request by mmcfarland**
#### RIP Original layer selector.  
This plugin has been removed as a core plugin, but is still available at
https://github.com/CoastalResilienceNetwork/map-tree and can be included
through a build process.  Additional necessary changes:
* Remove references is core region framework to layer_selctor
* Remove sample_plugin which showed how to extend layer_selector
* Move library `tv4`, which was included with layer_selector, but was
  refereced via `use!` by multiple other plugins, to the core framework.

Connects https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/653

#### Testing
* Verify the site loads and the tests pass
* Verify layer_selector is not found
* Add map tree into a `layer_selector` directory in `plugins/`, 
```
git clone git@github.com:CoastalResilienceNetwork/map-tree.git layer_selector
``` 
  and verify it still works

### CODE REVIEW

File: src/GeositeFramework/region.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "language": "en",
+    "language": "es",
```
Comment (caseypt) [inline src/GeositeFramework/region.json]: Leftover from debugging?

**Comment by caseypt** [comment]
I think we knew this would happen, but plugins that are dependent on the layer selector don't work once the layer selector is removed from the framework.

**Comment by caseypt** [comment]
I'm getting this error when trying to load the site:

image

**Comment by mmcfarland** [comment]
@caseypt I'm not getting that error.  Can you make sure you kill your appserver/hard refresh and see if you still get it?

**Comment by caseypt** [comment]
Yeah. I closed down Visual Studio, cleared the browser cache, etc.

This is what the `use` config object contains when it tries to find the config for `tv4`.

image

**Comment by caseypt** [comment]
I'm not totally sure what this is actually doing, but this change fixes it for me: 

```diff
diff --git a/src/GeositeFramework/Views/Home/Index.cshtml b/src/GeositeFramework/Views/Home/Index.cshtml
index 45db79b..2d6e2f3 100644
--- a/src/GeositeFramework/Views/Home/Index.cshtml
+++ b/src/GeositeFramework/Views/Home/Index.cshtml
@@ -594,6 +594,7 @@
         var useConfig = { @Html.Raw(Model.ConfigurationForUseJs) };
         useConfig['Azavea'] = { attach: 'Azavea' };
         useConfig['Geosite'] = { attach: 'Geosite' };
+        useConfig['tv4'] = { attach: 'tv4' };

         require({
             async: true,
```

**Comment by caseypt** [comment]
+1 (besides the language change). Tested with the `regional-planning` change. Looks good.

**Comment by mmcfarland** [comment]
Cool, removed the language setting change.  Good catch on that.

**Comment by rajadain** [comment]
+1 tested

**Pull request merged**